packages/main/src/components/DynamicPage/DynamicPage.jss.ts

```diff
@@ -0,0 +1,162 @@
+import { ThemingParameters } from '@ui5/webcomponents-react-base/lib/ThemingParameters';
+import { sapUiResponsiveContentPadding } from '@ui5/webcomponents-react-base/lib/spacing';
+
+export const DynamicPageCssVariables = {
+  anchorFloat: '--_ui5wcr_ObjectPage_actions_float',
+  anchorLeft: '--_ui5wcr_ObjectPage_actions_left',
+  anchorRight: '--_ui5wcr_ObjectPage_actions_right',
+  avatarMargin: '--_ui5wcr_ObjectPage_avatar_margin',
+  lastSectionMargin: '--_ui5wcr_ObjectPage_last_section_margin_bottom'
+};
+
+const styles = {
+  dynamicPage: {
+    width: '100%',
+    height: '100%',
+    maxHeight: '100vh',
```

Comment:
using 100vh is dangerous as we shouldn't make assumptions on where a dynamic page is used in an app. I would recommend using 100% as maxHeight